specification/api-metrics-user.md

```diff
@@ -9,9 +9,23 @@ TODO: Add a table of contents.
 
 ## Overview
 
-Metric instruments are the entry point for application and framework
-developers to instrument their code using counters, gauges, and
-measures.
+Metric instruments are the entry point for application and framework developers to instrument their code using counters, gauges, and measures.
+Metrics are created by calling methods on a `Meter` which is in turn created by a global `MeterFactory`.
+
+### Obtaining a Meter
+
+New `Meter` instances can be created via a `MeterFactory` and its `getMeter` method.
+`MeterFactory`s are generally expected to be used as singletons.
+Implementations SHOULD provide a single global default `MeterFactory`. The `getMeter` method expects two string arguments:
+
+- `name` (required): This name must identify the instrumentation library (also referred to as integration, e.g. `io.opentelemetry.contrib.mongodb`)
+and *not* the instrumented library.
+In case an invalid name (null or empty string) is specified, a working default `Meter` implementation as a fallback is returned
+rather than returning null or throwing an exception.
+A library implementing the OpenTelemetry API *may* also ignore this name and return a default instance for all calls
+if it does not support "named" functionality (e.g. an implementation which is not even observability-related).
```

Comment:
What is this trying to say? Just that applications cannot rely on different instances being returned for different names? Then this should be stated as such. Right now this sentence sounds too vague for a spec. State the requirements that implementations of this method must fulfill, or, equivalently, the properties of this method that  callers can rely on.